cmd/antrea-agent/agent.go

```diff
@@ -166,7 +166,8 @@ func run(o *Options) error {
 	// notifying NetworkPolicyController to reconcile rules related to the
 	// updated Pods.
 	podUpdates := make(chan v1beta1.PodReference, 100)
-	networkPolicyController := networkpolicy.NewNetworkPolicyController(antreaClientProvider, ofClient, ifaceStore, nodeConfig.Name, podUpdates)
+	AntreaPolicyConfig := features.DefaultFeatureGate.Enabled(features.AntreaPolicy)
```

Comment:
can we not use features to figure out whether it is enabled directly in agentcontroller  instead of passing the bool ?

Reply:
imported feature in networkpolicy_controller.go. Is it ok?